src/components/dialog/bl-dialog.css

```diff
@@ -21,9 +23,9 @@
   position: fixed;
 
   /* FIXME: Use z-index variable */
-  z-index: 999; 
+  z-index: 999;
```

Comment:
We can update those after z-index variable pr is merged

Reply:
This is an automatic change by my IDE, removing trailing spaces once I save a file.

Reply:
I pointed out because preventing having conflict between z-index pr and this actually. np.

Reply:
I'll fix the conflict, no problem.